cms/envs/test.py

```diff
@@ -325,6 +325,5 @@
 INSTALLED_APPS += ('openedx.core.djangoapps.ccxcon',)
 FEATURES['CUSTOM_COURSES_EDX'] = True
 
-# API access management. Necessary so that django-simple-history
-# doesn't break when running pre-test migrations.
+# API access management -- needed for simple-history to run.
```

Comment:
Basically, simple history tries to run a migration to create the history table, which expects any simple-history-having models to be in `INSTALLED_APPS`. Under CMS settings this app is not in `INSTALLED_APPS`, and breakage results which can be seen in the stacktrace I posted earlier. So the solution seems to be to install the app, then not run the tests.